recipes/libestools/meta.yaml

```diff
@@ -0,0 +1,60 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+# If your package is python based, we recommend using Grayskull to generate it instead:
+# https://github.com/conda-incubator/grayskull
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "libestools" %}
+{% set version = "2.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/festvox/speech_tools/archive/63ff01938f81443f5a294bc5f9ea6ac6ab38f6b0.tar.gz
```

Comment:
Do they have tags or publish release tar balls somewhere else?

Reply:
Eg. I have found these links: https://www.cstr.ed.ac.uk/downloads/festival/2.4/

Reply:
Unfortunately, no. No releases/tags on github. Their homepage contains a download section, but that contains only release 2.4 from 2014...